webbpsf/trending.py

```diff
@@ -628,18 +631,27 @@ def single_measurement_trending_plot(
         print('     Previous OPD is:', prev_filename)
 
     prev_opd, prev_opd_hdu = _read_opd(prev_filename)
+    prev_sensing_apername = prev_opd_hdu[0].header['APERNAME']
 
     if subtract_target:
         if verbose:
             print('     Subtracting NIRCam SI WFE target phase map')
 
-        # Get WSS Target Phase Map
-        was_targ_file = os.path.join(webbpsf.utils.get_webbpsf_data_path(), 'NIRCam', 'OPD', 'wss_target_phase_fp1.fits')
+        # Get WSS Target Phase Map for the sensing aperture
+        # Note that the sensing maintenance program changed field point from NRC A3 to A1 around Dec 2024.
+        was_targ_file = webbpsf.utils.get_target_phase_map_filename(sensing_apername)
+        prev_was_targ_file = webbpsf.utils.get_target_phase_map_filename(prev_sensing_apername)
+
+
         target_1024 = astropy.io.fits.getdata(was_targ_file)
         target_256 = poppy.utils.krebin(target_1024, (256, 256)) / 16  # scale factor for rebinning w/out increasing values
 
+        prev_target_1024 = astropy.io.fits.getdata(prev_was_targ_file)
```

Comment:
A small performance efficiency improvement would be to wrap this in a check for whether the target file is the same or not: 
```
    if prev_was_targ_file != was_targ_file:
        prev_target_1024 = astropy.io.fits.getdata(prev_was_targ_file)
        prev_target_256 = poppy.utils.krebin(prev_target_1024,
                                         (256, 256)) / 16  # scale factor for rebinning w/out increasing values
    else:
        prev_target_256 = target_256
```

That would save a bunch of redundant math for the common case of the prev and current phase maps being the same.